.github/workflows/testing.yml

```diff
@@ -63,3 +63,6 @@ jobs:
           COVERALLS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           make coveralls
+
+      - name: Run benchmark
+        run: make bench
```

Comment:
What is the goal? Ensure that all benchmarks are workable? Okay, but, please, reflect it in in the step name somehow and run just one iteration (to don't slowdown testing).

Reply:
By the way,

> -short
>
> Tell long-running tests to shorten their run time.
> It is off by default but set during all.bash so that installing
> the Go tree can run a sanity check but not spend time running
> exhaustive tests.

From https://pkg.go.dev/cmd/go#hdr-Testing_flags.

Reply:
Now I set the variable `DURATION=1x` to run all tests just one time.